src/com/grundszok/piotr/app/App.java

```diff
@@ -2,16 +2,16 @@
 
 import com.grundszok.piotr.app.controllers.MenuController;
 import com.grundszok.piotr.app.services.InputService;
+import com.grundszok.piotr.app.services.PersistanceService;
 import com.grundszok.piotr.app.services.display.AstrixDisplayService;
 import com.grundszok.piotr.app.services.display.DisplayService;
 
 class App {
     public static void main(String[] args) {
+        PersistanceService persistanceService = new PersistanceService();
         DisplayService displayService = new AstrixDisplayService();
         InputService inputService = new InputService(displayService);
-        MenuController menuController = new MenuController(displayService, inputService);
-
-        System.out.println("Git test");
+        MenuController menuController = new MenuController(displayService, inputService, persistanceService);
```

Comment:
Can be initialized directly in Menu Controller constructor

Reply:
Ok, but i will leave DisplayServivce as variable